Formula/liblinear.rb

```diff
@@ -29,8 +29,10 @@ class Liblinear < Formula
   def install
     system "make", "all"
     bin.install "predict", "train"
-    lib.install "liblinear.dylib"
-    lib.install_symlink "liblinear.dylib" => "liblinear.1.dylib"
+
+    liblinear = Dir[shared_library("liblinear", "*")].first
+    lib.install liblinear
+    lib.install_symlink liblinear => shared_library("liblinear")
```

Comment:
This won't work correctly unless we update patch https://github.com/Homebrew/formula-patches/blob/master/liblinear/patch-Makefile.diff#L30

The idea being to change `LIBEXT=".dylib";` to `LIBEXT=".$(SHVER).dylib";` so that the versioned shared library is installed first and an unversioned symlink is added.

Otherwise, it tries to create symlink with same name.